server/models/user.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const bcrypt = require("bcryptjs");
+
+const userSchema = new Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    unique: true,
+    required: true,
+  },
```

Comment:
I think we should use [indexing](https://stackoverflow.com/questions/36822937/purpose-of-index-in-mongoose-schema) here
```suggestion
  email: {
    type: String,
    index: { unique: true },
    required: true,
  },
```

Reply:
This user model is temporary, just for testing purpose. I shouldn't have committed it for now.